### DISCUSSION TITLE
Consider cluster.ws and wip.la to be TLDs in Vercel

### DISCUSSION THREAD

**Discussion by politician** [vercel/community#887] [Help]
I run a free service that lets developers create subdomains under cluster.ws and wip.la by opening a PR on https://github.com/Olivr/free-domain

Some users have reported not being able to configure them in Vercel, see the issue here: https://github.com/Olivr/free-domain/issues/208

I tried to [add both cluster.ws and wip.la to the public suffix list](https://github.com/publicsuffix/list/pull/1523) but it was rejected under the premise I should contact services directly instead, which I am now doing.

**Comment by amyegan** [comment]
Hi @politician! Sorry you're running into this problem. 

As @fu-sen mentioned in the linked issue https://github.com/Olivr/free-domain/issues/208, the TXT record is required as a way to protect custom domains already associated with another account. 

I'm going to ask the team and try to find out if there is any way around this. I'll update here when I know more.
